mongo/pagination/pagination_by_cursor.go

```diff
@@ -0,0 +1,102 @@
+package pagination
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Scalingo/go-utils/mongo/document"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func NewPaginationByCursorService(opts ServiceOpts) ServiceByCursor {
```

Comment:
You are in a package named `pagination`. You don't need to add the word "pagination" in the method or structure name. Can you remove it where it's needed?
```suggestion
func NewByCursorService(opts ServiceOpts) ServiceByCursor {
```